sci-libs/gdcm/gdcm-3.0.14.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_9 )
+
+inherit cmake python-single-r1
+
+DESCRIPTION="Cross-platform DICOM implementation"
+HOMEPAGE="http://gdcm.sourceforge.net/"
+SRC_URI="mirror://sourceforge/gdcm/${P}.tar.bz2
+	test? ( mirror://sourceforge/gdcm/gdcmData.tar.gz )" # 3.0.14: .bz2 is broken, should be checked in next release
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	app-text/poppler:0=[cxx]
+	dev-libs/expat:0=
+	dev-libs/json-c:0=
+	dev-libs/libxml2:2=
+	dev-libs/openssl:0=
+	>=media-libs/charls-2.0.0:0=
+	>=media-libs/openjpeg-2.0.0:2=
+	sys-apps/util-linux:0=
+	sys-libs/zlib:0=
+	vtk? ( sci-libs/vtk[rendering,python?] )
+	vtk? ( python? ( <sci-libs/vtk-9 ) )
```

Comment:
```suggestion
	vtk? ( sci-libs/vtk[rendering]
        	python? ( 
        	       <sci-libs/vtk-9 [python,${PYTHON_SINGLE_USEDEP}]
        	       ${PYTHON_DEPS}
        	) 
        )
```

Reply:
> Same applies to sci-libs/itk with vtkglue/python (missing PYTHON_SINGLE_USEDEP).

I noticed that you marked the conversations as resolved, but the ebuild still is the same. Did you perhaps forget to push your changes?